napari/_qt/qt_viewer.py

```diff
@@ -461,6 +469,14 @@ def _create_canvas(self) -> None:
 
         self.canvas.destroyed.connect(self._diconnect_theme)
 
+    def _throttle_mouse_event(self, event=None):
+        self._throttled_event = event
+        self._mouse_throttler.throttle()
+
+    def _fire_throttled_mouse_event(self):
+        if self._throttled_event is not None:
+            self.on_mouse_move(self._throttled_event)
```

Comment:
```suggestion
        if self._throttled_event is not None:
        	event = self._throttled_event
        	self._throttled_event = None
            self.on_mouse_move(event)
```

maybe a clean variable?

Reply:
Actually, all of this is probably breaking terribly with threading...

Reply:
Throttler is implemented using a timer, so everything should be in the main thread. 
My proposition is not for threading, but to prevent double calls on the same event.

Reply:
I just replaced the whole thing with a one-liner that handles everything, after a quick chat with @tlambert03 :P

Reply:
But the solution is not correct... (qt specific problem)

I put an explanation in a separate comment